### PR TITLE
EZP-28773: [v2] Issues with page scrolling during actions in rich text editor

### PR DESCRIPTION
**Pull request by konradoboza**
| Question      | Answer
| ------------- | ---
| Tickets       | https://jira.ez.no/browse/EZP-28773
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Tests pass?   | yes
| Doc needed?   | no
| License       | [GPL-2.0](https://github.com/ezsystems/ezplatform-admin-ui/blob/master/LICENSE)

This PR contains fixes ported from v1 for all reported issues regarding scrolling when inserting objects/links into the RichText.

#### Checklist:
- [x] Coding standards (`$ composer fix-cs`)
- [x] Ready for Code Review

### CODE REVIEW

File: src/bundle/Resources/public/js/alloyeditor/src/buttons/ez-btn-embed.js [hunk 1]
```diff
@@ -27,8 +27,15 @@ export default class EzBtnEmbed extends EzEmbedDiscoverContentButton {
      */
     addEmbed(items) {
         const contentInfo = items[0].ContentInfo.Content._id;
+        let scrollY;
 
-        this.execCommand();
+        if (navigator.userAgent.indexOf('Chrome') > -1) {
+            scrollY = window.pageYOffset;
```
Comment (dew326) [inline src/bundle/Resources/public/js/alloyeditor/src/buttons/ez-btn-embed.js]: `const scrollY = window.pageYOffset;` and the `let` is not needed (applies to all cases).

File: src/bundle/Resources/public/js/alloyeditor/src/buttons/ez-btn-linkedit.js [hunk 2]
```diff
@@ -386,14 +393,35 @@ export default class EzBtnLinkEdit extends Component {
...
     }
+
+    /**
+     * Saves current scrollbar position, invokes callback function and scrolls
+     * to the saved position afterward.
+     *
+     * @method restoreScrollbarPosition
+     * @param {Function} callback invoked after saving current scrollbar position
+     */
+    restoreScrollbarPosition(callback) {
```
Comment (dew326) [inline src/bundle/Resources/public/js/alloyeditor/src/buttons/ez-btn-linkedit.js]: I don't like the name of this method. Not really `restore`. 
Maybe something like `invokeWithFixedScrollbar`?

**Comment by konradoboza** [comment]
@dew326 fixed.

**Comment by micszo** [comment]
Indeed because fix for tests was only done from 1.3 (https://github.com/ezsystems/ezplatform-admin-ui/commit/f86558e4a5695582ada32efe08b3d294c5ad5793).

**Comment by lserwatka** [comment]
@m-tyrala behat is failing here. Is it related?

**Comment by m-tyrala** [comment]
@lserwatka looking into it, there is an error in PHP Unit tests not in Behat tests

> 1) EzSystems\EzPlatformAdminUi\Tests\Form\DataTransformer\LocationsTransformerTest::testReverseTrans
> formWithIds
> Failed asserting that two arrays are equal.
> --- Expected
> +++ Actual
> @@ @@
>  Array (
> -    0 => eZ\Publish\Core\Repository\Values\Content\Location Object (...)
> -    1 => eZ\Publish\Core\Repository\Values\Content\Location Object (...)
> +    0 => null
> +    1 => null
> /home/travis/build/ezsystems/ezplatform-admin-ui/src/lib/Tests/Form/DataTransformer/LocationsTransformerTest.php:48

**Comment by konradoboza** [comment]
As I mentioned earlier, it doesn't seem related.

**Comment by lserwatka** [comment]
You can merge it up

**Pull request merged**